binding.cc

```diff
@@ -90,6 +94,48 @@ void ExitOnPromiseRejectCallback(PromiseRejectMessage promise_reject_message) {
   exit(1);
 }
 
+MaybeLocal<Module> ResolveCallback(Local<Context> context,
+                                   Local<String> specifier,
+                                   Local<Module> referrer) {
+  auto isolate = Isolate::GetCurrent();
+  worker* w = (worker*)isolate->GetData(0);
+
+  HandleScope handle_scope(isolate);
+
+  String::Utf8Value str(specifier);
+  char* moduleName = *str;
```

Comment:
I would make these variables `const` because there's no reason not to.

Reply:
I had an error transferring the `const` to Go - I have very minimal experience here so any suggestions on how to get them into Go would be helpful.

Reply:
Got it. Don't know how to interop with Go on this one, sorry. =)